torch/_ops.py

```diff
@@ -16,6 +16,9 @@
 from torch.utils._python_dispatch import TorchDispatchMode
 
 
+F = TypeVar("F", bound=Callable[..., Any])
```

Comment:
it's probably a private Type variable in which case it and it's string representation should start with underscore to prevent it from being exported